src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -573,32 +621,154 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
+		app.setLaunchArgument(LaunchArguments.healthCertificate.firstAndSecondHealthCertificate, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.isDCCAdmissionCheckScenariosEnabled, to: true)
+		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
+		app.launch()
+		
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap()
+		
+		// Check if the selection button exists
+		let selectionButtonCell = try XCTUnwrap(app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.changeAdmissionScenarioCell])
+		
+		// Navigate to the scenario selection screen
+		selectionButtonCell.waitAndTap()
+		
+		// Select federal rules
+		app.cells.element(boundBy: 0).waitAndTap()
+		
+		// check the Mask State.
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell].exists)
+		XCTAssertTrue(app.otherElements[AccessibilityIdentifiers.HealthCertificate.MaskState.roundedView].exists)
+		
+		snapshot("screenshot_mask_optional_certificate_overview_federal_rules")
+	}
 
-	func test_AdmissionStateChanges_Then_StateChangeIndicatorIsVisible() {
+	func test_screenshot_WhenMaskIsRequired_DetailScreen() throws {
 		app.setLaunchArgument(LaunchArguments.healthCertificate.firstAndSecondHealthCertificate, to: true)
 		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
 		app.setLaunchArgument(LaunchArguments.healthCertificate.testCertificateRegistered, to: true)
 		app.launch()
-
+		
 		// Navigate to Certificates Tab.
-		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap(.long)
-
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap(.medium)
+		
 		// Navigate to Persons Tab.
-		app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell].waitAndTap(.long)
-
-		XCTAssertTrue(app.otherElements[AccessibilityIdentifiers.HealthCertificate.AdmissionState.unseenNewsIndicator].waitForExistence(timeout: .long))
+		app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell].waitAndTap()
+		
+		// check the Mask State.
+		XCTAssertTrue(app.otherElements[AccessibilityIdentifiers.HealthCertificate.MaskState.roundedView].exists)
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.HealthCertificate.MaskState.title].exists)
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.HealthCertificate.MaskState.subtitle].exists)
+		XCTAssertTrue(app.images[AccessibilityIdentifiers.HealthCertificate.MaskState.badgeImage].exists)
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.HealthCertificate.MaskState.description].exists)
+		XCTAssertTrue(app.links[AccessibilityIdentifiers.HealthCertificate.MaskState.faq].exists)
+		
+		snapshot("screenshot_mask_required_certificate_detail_part1")
+		
+		app.swipeUp(velocity: .fast)
+		
+		snapshot("screenshot_mask_required_certificate_detail_part2")
```

Comment:
Why do we need this?

Reply:
to take the screenshot of the complete screen. So, we need to scroll the screen.

Reply:
Was it required? Because what I see from the title is we are testing the maskState only.

Reply:
Not sure but I am just trying to capture the certificate at the bottom. If there is no need then doesn't make sense.